src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -493,25 +537,50 @@ export class HaAutomationEditor extends KeyboardShortcutMixin(LitElement) {
     history.back();
   }
 
-  private async _handleMenuAction(ev: CustomEvent<ActionDetail>) {
-    switch (ev.detail.index) {
-      case 0:
-        this._mode = "gui";
-        break;
-      case 1:
-        this._mode = "yaml";
-        break;
-      case 2:
-        this._duplicate();
-        break;
-      case 3:
-        this._deleteConfirm();
-        break;
+  private _switchUiMode() {
+    this._mode = "gui";
+  }
+
+  private _switchYamlMode() {
+    this._mode = "yaml";
+  }
+
+  private async promptAutomationAlias(): Promise<string | null> {
```

Comment:
```suggestion
  private async _promptAutomationAlias(): Promise<string | null> {
```

Reply:
Changed in [03a680e](https://github.com/home-assistant/frontend/pull/13488/commits/03a680ea469a0e02050ee58bd2ffd2c9802cc0c3)